Src/CZICmd/BitmapGenNull.cpp

```diff
@@ -215,7 +215,7 @@ CBitmapGenNull::CBitmapGenNull()
 
 }
 
-/*virtual*/std::shared_ptr<libCZI::IBitmapData> CBitmapGenNull::Create(libCZI::PixelType pixeltype, std::uint32_t width, std::uint32_t height, const BitmapGenInfo& info)
```

Comment:
Instead of removing the identifier names for unused arguments, I'd rather have them commented out. Thus, keeping the names in the source for documentation.
But well - I'd be fine with omitting them as well.
C++17 would add `[maybe_unused]`, but so far we require only C++14.

Reply:
I went for the traditional way